src/workloads/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,1224 @@
+            cursor: {}
+
+- Name: ExpressionCompare
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionCompare
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {cmp: { $cmp: ["$a", 0]}}}]
+            cursor: {}
+
+- Name: ExpressionConcat
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionConcat
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {concat: { $concat: ["$string", "_", "$string2"]}}}]
+            cursor: {}
+
+- Name: ExpressionConcatArrays
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionConcatArrays
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {concatArrays: { $concatArrays: ["$array1", "$array2"]}}}]
+            cursor: {}
+
+- Name: ExpressionCond
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionCond
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {cond: { $cond: { if: { $gte: ["$int", 0] }, then: 100, else: -100 } }}}]
+            cursor: {}
+
+- Name: ExpressionDateDiff
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionDateDiff
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {dateDiff: {$dateDiff: {startDate: "$date1", endDate: "$date2", unit: "millisecond"}}}}]
+            cursor: {}
+
```

Comment:
I did another pass through https://docs.google.com/spreadsheets/d/1Nnt_vqG3uC4UTft4Eunuf5iq2eLsx9Eo1LRY2kJ5BGE/edit#gid=0 and found some more expressions that need perf coverage; these are highlighted in the last column in bright pink and they are:
$last
$dateFromParts
$dateToParts
$subtract

Could you add these? Sorry I didn't catch these sooner!

Reply:
Done.